lib/travis/api/v3/renderer/repository.rb

```diff
@@ -1,7 +1,7 @@
 module Travis::API::V3
   class Renderer::Repository < ModelRenderer
     representation(:minimal,  :id, :name, :slug)
-    representation(:standard, :id, :name, :slug, :description, :github_language, :active, :private, :owner, :default_branch, :starred)
+    representation(:standard, :id, :name, :slug, :description, :github_language, :active, :private, :owner, :default_branch, :starred, :managed_by_installation, :active_on_org)
```

Comment:
From my reading of [the comments](https://github.com/travis-pro/team-teal/issues/2608#issuecomment-377161126), the goal was to expose `locked` via the API rather than `active_on_org`.

You can add such methods directly to the renderer:

``` ruby
module Travis::API::V3
  representation(:standard, ..., :managed_by_installation, :locked, :locked_reason)

  def managed_by_installation
    model.managed_by_installation?
  end

  def locked
    model.active_on_org?
  end

  def locked_reason
    'active_on_org' if model.active_on_org?
  end
end
```

Reply:
we discussed this and decided that `locked` wasn't a good name. This is probably not reflected in the docs...